site/src/App/routes/guides/playroom-prototyping.tsx

```diff
@@ -30,14 +39,172 @@ const PlayroomPrototyping = () => (
+      <TextLink href="https://www.sketch.com/docs/designing/symbols">
+        Sketch symbol,
+      </TextLink>{' '}
+      only much more powerful.
+    </Text>
+    <Notice tone="info">
+      <Text>
+        To keep the code clean, we’ll store our job data in a piece of state
+        called <Strong>&ldquo;jobs&rdquo;</Strong>. We’ll cover the concept of
+        state in more detail later.
+      </Text>
+    </Notice>
+    <Code>
+      {({ setDefaultState, getState }) => {
+        const result = source(
+          <>
+            {setDefaultState('jobs', [
+              {
+                id: 1,
+                title: 'Lead Designer',
+                location: 'Melbourne',
+              },
+              {
+                id: 2,
+                title: 'Senior Developer',
+                location: 'Sydney',
+              },
+              {
+                id: 3,
+                title: 'Product Manager',
+                location: 'Canberra',
+              },
+            ] as Array<MockJob>)}
+
+            <Stack space="medium">
+              {getState('jobs').map((job: MockJob) => (
+                <Card key={job.id}>
+                  <Stack space="small">
+                    <Text weight="strong">{job.title}</Text>
+                    <Text>{job.location}</Text>
+                  </Stack>
+                </Card>
+              ))}
+            </Stack>
+          </>,
+        );
+
+        return {
+          ...result,
+          code: result.code
+            .replace(/ as Array<MockJob>/g, '')
+            .replace(/: MockJob/g, ''),
+        };
+      }}
+    </Code>
+    <Text>
+      We now have the same design as before, but expressed in a way that makes
+      it much easier for us to change. Any updates we make to this card will now
+      affect every card on the screen. It’s also much faster for us to make
+      changes to the data without having to touch the UI.
+    </Text>
+    <Text>
+      Your data can also contain optional fields. For example, we may want to
+      provide an optional <Strong>featured</Strong> property that toggles the
+      visibility of a badge.
+    </Text>
+    <Code>
+      {({ setDefaultState, getState }) => {
+        const result = source(
+          <>
+            {setDefaultState('jobs', [
+              {
+                id: 1,
+                title: 'Lead Designer',
+                location: 'Melbourne',
+              },
+              {
+                id: 2,
+                featured: true,
+                title: 'Senior Developer',
+                location: 'Sydney',
+              },
+              {
+                id: 3,
+                title: 'Product Manager',
+                location: 'Canberra',
+              },
+            ] as Array<MockJob>)}
+
+            <Stack space="medium">
+              {getState('jobs').map((job: MockJob) => (
+                <Card key={job.id}>
```

Comment:
As well as conditional nodes, thoughts on demoing a conditional prop value here too? For example a `tone` on `Card`?
```suggestion
                <Card key={job.id} tone={job.featured && 'promote'}>
```